### PR TITLE
Initial implementation of the KubeOne website

### PR DESCRIPTION
**Pull request by xmudrii**
Initial implementation of the KubeOne website.

/assign @kron4eg

### CODE REVIEW

**Comment by scheeles** [comment]
I think the `development` part can stay at KubeOne repo

**Comment by xmudrii** [comment]
@scheeles Done.

File: content/kubeone/_index.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
++++
+title = "Docs"
```
Comment (kron4eg) [inline content/kubeone/_index.md:2]:
```suggestion
title = "KubeOne Docs"
```

File: content/kubeone/master/_index.en.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
++++
+title = "Welcome"
```
Comment (kron4eg) [inline content/kubeone/master/_index.en.md:2]:
```suggestion
title = "KubeOne documentation in development version"
```

File: content/kubeone/master/getting_started/_index.en.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
++++
+title = "Getting Started"
```
Comment (kron4eg) [inline content/kubeone/master/getting_started/_index.en.md:2]:
```suggestion
title = "Getting Started with KubeOne"
```

File: content/kubeone/master/getting_started/frequently_asked_questions/_index.en.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
++++
+title = "Frequently Asked Questions"
```
Comment (kron4eg) [inline content/kubeone/master/getting_started/frequently_asked_questions/_index.en.md:2]:
```suggestion
title = " KubeOne FAQ | Frequently Asked Questions"
```

File: content/kubeone/master/getting_started/frequently_asked_questions/_index.en.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
++++
```
Comment (kron4eg) [inline content/kubeone/master/getting_started/frequently_asked_questions/_index.en.md:1]: I think this whole file can be moved one level up

File: data/version.yaml [hunk 6]
```diff
@@ -7,10 +7,10 @@ kubermatic:
...
-#     - release: master
-#       name: master
-#     - release: v0.12
-#       name: v0.12
+kubeone:
+  name: KubeOne
+  versions:
+    - release: master
+      name: master
+    - release: v0.11
```
Comment (kron4eg) [inline data/version.yaml:15]: probably for now we can avoid 0.11 (as we anyway don't have its copy), at new release time we'd start providing 1.0 dedicated docs and onward

**Comment by xmudrii** [comment]
@kron4eg Comments addressed.

**Pull request merged**